src/components/Timelog/TimeEntryForm/TimeEntryForm.jsx

```diff
@@ -112,29 +113,38 @@ const TimeEntryForm = (props) => {
   useEffect(() => {
     axios
       .get(ENDPOINTS.TASKS_BY_USERID(userId))
-      .then((res) => {
+      .then(res => {
         setTasks(res?.data || []);
       })
       .catch(err => console.log(err));
   }, []);
 
+  //grab form data before editing
+  useEffect(() => {
+    if (isOpen && edit) {
+      setFormDataBeforeEdit(inputs);
+    } else {
+      setFormDataBeforeEdit({});
```

Comment:
is it necessary to set setFormDataBefore to empty object every time that isOpen or edit is false?

Reply:
Thanks for pointing it out, it's not necessary to write else condition, I'm gonna change it.